app/views/pilots/index.html.haml

```diff
@@ -16,18 +16,23 @@
                 %th{scope: 'col', width: '10%'}= 'Flights'
                 %th{scope: 'col', width: '10%'}= 'Hours'
                 %th{scope: 'col', width: '10%'}= 'Last Flight'
+                %th{scope: 'col', width: '10%'}
 
             %tbody
               - if @pilots.empty?
                 %tr
                   %td.text-muted{colspan: 20}= :None
               - @pilots.each do |pilot|
                 %tr
-                  %td.align-middle= link_to pilot.pid_to_s, pilot_logbook_path(pilot)
+                  %td.align-middle= pilot.pid_to_s
                   %td.align-middle= pilot.full_name
                   %td.align-middle= number_with_delimiter(pilot.total_flights)
                   %td.align-middle= number_with_delimiter(pilot.total_hours)
                   %td.align-middle= pilot_last_flight(pilot)
+                  %td.align-middle
+                    = link_to pilot_logbook_path(pilot), class: "btn btn-primary btn-sm" do
+                      %i.fas.fa-search
+                      More Info
```

Comment:
"More Info" might be a bit large for a button that is repeated often in a table such as this. Additionally it might not render very friendly on small resolutions (phones/tablets)

My vote would be a single word like "View" or even just a button with an icon may be more aesthetically acceptable.

Reply:
Added "View" for now. If there had been more than one button I would be more inclined to go with just the icon. Think it may be too vague initially seeing as we do have enough space for "View"
Commited and pushed!

Reply:
Agreed. I think this is a good solution.